lua/nvim-lsp-installer/servers/wgsl_analyzer/init.lua

```diff
@@ -0,0 +1,32 @@
+local server = require "nvim-lsp-installer.server"
+local cargo = require "nvim-lsp-installer.core.managers.cargo"
+local lspconfig = require "lspconfig"
+local configs = require "lspconfig.configs"
+
+if not configs.wgsl_analyzer then
+    configs.wgsl_analyzer = {
+        default_config = {
+            cmd = { "wgsl_analyzer" },
+            filetypes = { "wgsl" },
+            root_dir = lspconfig.util.root_pattern(".git", "wgsl"),
```

Comment:
I'm not familiar with WGSL, but this root dir looks a bit weird to me - is there commonly a file called `wgsl` in the root of WGSL projects?